tests/functional/graphql/01-workflow.t

```diff
@@ -96,7 +96,7 @@ cmp_json "${TEST_NAME}-out" "${TEST_NAME_BASE}-workflows.stdout" << __HERE__
             "newestActiveCyclePoint": "20210101T0000Z",
             "oldestActiveCyclePoint": "20210101T0000Z",
             "reloaded": false,
-            "runMode": "live",
+            "runMode": "Live",
```

Comment:
This is the case change that Tim mentioned above.

I'm pretty sure we can avoid this case change which would make life easier elsewhere in the codebase.

I think this might be the only schema change in this PR (the rest are graphene backend changes that don't actually change the generated GraphQL schema which is good news), so we might be able to remove the "schema change" label if we resolve this.

Reply:
> I'm pretty sure we can avoid this case change which would make life easier elsewhere in the codebase.

What just for the `Workflow.run_mode` turn that back to a String type?

What about `Runtime.run_mode` do you want to stop using enums for that too?
Would that mean we don't have the fancy Edit Runtime menu?
![image](https://github.com/user-attachments/assets/9a546591-9a48-4778-af85-00a849f1bab2)

Or do we want `"runMode": "Live"` for tasks and `"runMode": "live"` for workflow (like we have currently)

Seems a little inconsistent.... I didn't make these changes on an "unnecessary" whim..


> I think this might be the only schema change

Well, functionally, yes..

Reply:
We don't need to stop using enums, just change the capitalisation of the enum keys.

Reply:
> We don't need to stop using enums. Just remove the .capitalize added in this PR.

We don't need to stop using the `RunMode` enum for *`Runtime.run_mode`*, however we do need to stick with string (not enum) for `Workflow.run_mode` - see https://github.com/cylc/cylc-flow/pull/6478#discussion_r2001342636

Reply:
done
![image](https://github.com/user-attachments/assets/e3c9d70b-5ac4-4078-be79-a22832f49954)